Streetcode/Streetcode.WebApi/Controllers/Partners/PartnersController.cs

```diff
@@ -45,7 +45,7 @@ public async Task<IActionResult> Create([FromBody] CreatePartnerDTO partner)
     }
 
     [HttpPut]
-    [AuthorizeRoles(UserRole.MainAdministrator, UserRole.Administrator)]
+    /*[AuthorizeRoles(UserRole.MainAdministrator, UserRole.Administrator)]*/
```

Comment:
Is that supposed to be removed or got included accidently?

Reply:
It will be removed when all work correctly, so it is now for easier use and testing.